.github/workflows/fw_icu4c_ci.yml

```
@@ -77,3 +108,19 @@ jobs:
           cd test/cintltst;
           echo 'Running ICU4C cintltst';
           make -j2 check;
+
+  debian-packaging:
+    needs: set-version-number
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Checkout ci-build-tools
+        run: git clone https://github.com/sillsdev/ci-builder-scripts
+
+      - name: Build debian package
+        run: |
```

I suggest the following.
```suggestion
        run: |
          set -xueo pipefail
```
Using `set -eo pipefail` means "If there is a problem, don't ignore it by continuing to process the rest of the script, but stop and exit instead.".
Using `set -u` means "If I try to use a variable (in bash) that isn't defined, it's a mistake so consider it an error".
Using `set -x` prints the commands being run. That adds more output, but it's nice to see what the commands are when you're diagnosing what didn't work.

---

👍